docroot/themes/humsci/humsci_basic/src/js/shared/carousel-slides/carousel-slides-height.js

```diff
@@ -0,0 +1,52 @@
+const slides = document.querySelectorAll('.paragraph--type--hs-carousel');
+let timeOutFunctionId; // a numeric ID which is used by clearTimeOut to reset the timer
+
+if (slides.length > 0) {
+  restrictHeight();
+
+  // Watch for when the browser window resizes, then run the restrictHeight
+  // function to reset the height of the text boxes
```

Comment:
I added a lot of comments thinking of how I could make it easier for Ryan to review. Please let me know if there are any that I should remove for being too verbose.